licenses/lzf.txt

```diff
@@ -0,0 +1,33 @@
+Copyright Notice and Statement for LZF filter
```

Comment:
I wonder now if we have to include licenses for lzf/blosc/h5py: because we no longer include C sources for lzf/blosc (which, I think, was the case early on) - as we rely on blosc-sys and lzf-sys.

Reply:
I think the rework of automatic chunking also means we now no longer have any direct code from `h5py`. We also comply with the `hdf5` license by including this in the `hdf5-src` archive. We should be safe to remove `licenses` (but I am not a lawyer).

Reply:
Yes, I think they are safe to remove now.